### PR TITLE
Improve new Pdo 8.4 changes

### PR DESCRIPTION
**Pull request by kamil-tekiela**
@Girgias  What do I need to do to make the documenation compile again locally. I keep getting this:

```
ERROR (file:///D:/projects/en/reference/pdo_pgsql/pdo-pgsql.xml:193:42)
 &reference.pdo-pgsql.pdo.entities.pgsql;
------------------------------------------^
 Entity 'reference.pdo-pgsql.pdo.entities.pgsql' not defined
```

### CODE REVIEW

**Comment by kamil-tekiela** [comment]
Should the new method be added here instead? https://www.php.net/manual/en/migration84.new-functions.php

**Comment by Girgias** [comment]
> Should the new method be added here instead? https://www.php.net/manual/en/migration84.new-functions.php

I don't think we mention new methods for new classes, as this is kinda implied.

**Comment by Girgias** [comment]
> @Girgias What do I need to do to make the documenation compile again locally. I keep getting this:
> 
> ```
> ERROR (file:///D:/projects/en/reference/pdo_pgsql/pdo-pgsql.xml:193:42)
>  &reference.pdo-pgsql.pdo.entities.pgsql;
> ------------------------------------------^
>  Entity 'reference.pdo-pgsql.pdo.entities.pgsql' not defined
> ```

Did you fetch the latest `doc-base` ?

**Comment by kamil-tekiela** [comment]
> > @Girgias What do I need to do to make the documenation compile again locally. I keep getting this:
> > ```
> > ERROR (file:///D:/projects/en/reference/pdo_pgsql/pdo-pgsql.xml:193:42)
> >  &reference.pdo-pgsql.pdo.entities.pgsql;
> > ------------------------------------------^
> >  Entity 'reference.pdo-pgsql.pdo.entities.pgsql' not defined
> > ```
> 
> Did you fetch the latest `doc-base` ?

Yes, I did as I always do. `git fetch upstream && git rebase upstream/master`
I don't know where this entity should be defined.

File: appendices/migration84/new-features.xml [hunk 1]
```diff
@@ -354,17 +354,17 @@ $object = $reflector->newLazyGhost($initializer);
...
-   of the driver-specific subclass directly.
+   <methodname>PDO::connect</methodname> method or by instantiating one
+   of the driver-specific subclasses directly.
   </simpara>
 
   <!-- RFC: https://wiki.php.net/rfc/pdo_driver_specific_parsers -->
   <para>
-   Added support for driver specific SQL parsers.
+   Added support for driver specific SQL parsers. When a driver-specific parser
+   is not available, the default parser is used.
```
Comment (Girgias) [inline appendices/migration84/new-features.xml:367]: Semantic line break
```suggestion
   Added support for driver specific SQL parsers.
   When a driver-specific parser is not available, the default parser is used.
```

**Review by Girgias — COMMENTED** [review]
I don't know why you can't build... it works just fine for me, but it seems like some entity is not being generated for you by configure.

**Review by Girgias — APPROVED** [review]

**Pull request merged**